modules/common-attributes.adoc

```diff
@@ -1,4 +1,5 @@
 {product-author}
+// '{product-title}' and '{product-version}' are parsed when AsciiBinder queries the '_distro_map.yml' file in relation to the base branch of a pull request.
```

Comment:
I am confused. `{product-version}` is defined on the next line.

```suggestion
// The {product-title} attribute provides the context-sensitive name of the relevant OpenShift distribution.
// Both the '{product-title}' and '{product-version}' attributes are parsed when AsciiBinder queries the '_distro_map.yml' file for a particular distribution in relation to the base branch of a pull request.
```

Reply:
+1, but I'd also give example:

```
// The {product-title} attribute provides the context-sensitive name of the relevant OpenShift distribution, for example, "OpenShift Container Platform" or "OKD".
```

I'm also unsure of what {product-author} and {product-version} are doing in this file, because it's not defining them as is, to my knowledge

Reply:
@bergerhoffer thank you for the suggestion. I have subsequently applied your example and included an example for `{product-version}` also.

@vikram-redhat and I were discussing the presence of `{product-author}` and `{product-version}` in this file earlier and it is being reviewed.

Reply:
My understanding is the same, `{product-version}` is not defined in this file.

Reply:
Following a conversation with @vikram-redhat, I am removing `{product-author}` and `{product-version}` from `modules/common-attributes.adoc` in this PR.